engine/client/cl_parse.c

```diff
@@ -2132,7 +2132,11 @@ void CL_ParseServerMessage( sizebuf_t *msg, qboolean normal_message )
 			old_background = cl.background;
 			if( MSG_ReadOneBit( msg ))
 			{
-				cls.changelevel = true;
+				// if it's local client, do not clean states on serverdata packet
```

Comment:
This is just wrong.

1. This branch always gets hit, even if it's not in fact a change level. See server code, when svc_changing message is sent.
2. Entities still need to be cleaned due to possible side-effects in interpolation code.

Reply:
Yes, this bug is still happens when playing on old co-op servers